types/forge-viewer/index.d.ts

```diff
@@ -263,6 +263,8 @@ declare namespace Autodesk {
           loadOptions?: object | undefined;
           sharedPropertyDbPath?: string | undefined;
           ids?: string | undefined;
+          applyScaling?: string;
```

Comment:
applyScaling is either string or object ({ from: 'ft', to: 'm' })

Reply:
Great! I took the liberty to type it exactly that way.